.github/workflows/doc-build.yml

```diff
@@ -26,7 +26,7 @@ on:
 
 env:
   # NOTE: west docstrings will be extracted from the version listed here
-  WEST_VERSION: 0.14.0
+  WEST_VERSION: 1.0.0a1
```

Comment:
a1? alpha1?

Reply:
Yes; this resolves a chicken-and-egg problem: we don't want to cut the release without working release notes and documentation, but we have to have a version up somewhere to pip install for the documentation build. So we cut an alpha, because pip ignores alpha versions unless explicitly requested, so it doesn't affect users until we have the docs ready to go.